glfw/src/Window.zig

```diff
@@ -1610,6 +1600,16 @@ pub const InputMode = enum(c_int) {
     sticky_mouse_buttons = c.GLFW_STICKY_MOUSE_BUTTONS,
     lock_key_mods = c.GLFW_LOCK_KEY_MODS,
     raw_mouse_motion = c.GLFW_RAW_MOUSE_MOTION,
+    
+    pub fn Value(comptime input_mode: InputMode) type {
```

Comment:
```suggestion
    /// Tells which value type is required for the given input mode when invoking `setInputMode`.
    /// Generally, you should use one of the `setInputMode` variants which does not take a `mode`
    /// parameter instead.
    pub fn Value(comptime input_mode: InputMode) type {
```

Reply:
The more I look at this, the restriction we impose on having `input_mode` be `comptime`, and the fact that we do have the typed variants... I wonder: should `setInputMode` actually be public?

The original thinking was that at least it allowed for someone to port some existing GLFW code over that maybe stores the input mode constant at runtime, but now that we're eliminating that possibility and it's only a comptime thing.. hmm.

Reply:
Maybe we should make it private, and if anyone complains about not having the ability to do a runtime `setInputMode` / having to write their own code to do that, we re-evaluate?

Reply:
Well, the other possibility here is my second proposition, "We assert that 'value' has the correct type based on the value of 'mode' at runtime", which has the same effect of allowing us to eliminate 'InvalidEnum', and the added clause that it would allow easier porting of code like you say.

Reply:
Yes, true, I guess that does make more sense. Sorry for not catching that earlier, I'm quite under the weather atm so my mind is a bit foggy.

Reply:
Aye, no worries mate, I know the feeling. That's why we have these review discussions at the end of the day; just gotta get on the same page.